<?xml version="1.0" ?>
<sdf version="1.5">
  <model name="obstacle_2">
    <static>true</static>
    <link name="link">
      <pose>0 0 0 0 0 0</pose>
      <collision name="collision">
        <geometry>
          <box>
            <size>1.0 0.5 5.0</size>
          </box>
        </geometry>
      </collision>
      <visual name="visual">
        <geometry>
          <box>
            <size>1.0 0.5 5.0</size>
          </box>
        </geometry>        
      </visual>
    </link>
  </model>
</sdf>
